### PR TITLE
Small change to DiagnosticFormatter

### PR DESCRIPTION
**Pull request by AdamSpeight2008**
Remove the String.Format, to use String Concatenation instead.
Simplify the Instance property,

### CODE REVIEW

**Comment by AdamSpeight2008** [comment]
@dotnet-bot retest this please.

**Comment by jcouv** [comment]
@AdamSpeight2008 What's the benefit of String.Concatenation here?

**Comment by AdamSpeight2008** [comment]
@jcouv Doesn't invoke String.Format. Few allocations.

File: src/Compilers/VisualBasic/Portable/Errors/DiagnosticFormatter.vb [hunk 1]
```diff
@@ -14,12 +14,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
...
         Friend Overrides Function FormatSourceSpan(span As LinePositionSpan, formatter As IFormatProvider) As String
-            Return String.Format("({0}) ", span.Start.Line + 1)
+            Return "(" & span.Start.Line + 1 &") "
         End Function
 
         ''' <summary>
         ''' Gets the current DiagnosticFormatter instance.
         ''' </summary>
-        Public Shared Shadows ReadOnly Property Instance As VisualBasicDiagnosticFormatter = New VisualBasicDiagnosticFormatter()
+        Public Shared Shadows ReadOnly Property Instance As New VisualBasicDiagnosticFormatter()
```
Comment (jmarolf) [inline src/Compilers/VisualBasic/Portable/Errors/DiagnosticFormatter.vb]: How is this related to reducing allocations by using string concatenation over `String.Format`?

**Comment by jcouv** [comment]
LGTM
@AlekseyTs for a second review?

File: src/Compilers/VisualBasic/Portable/Errors/DiagnosticFormatter.vb (hunk 1 above)
Comment (AdamSpeight2008) [inline src/Compilers/VisualBasic/Portable/Errors/DiagnosticFormatter.vb]: It doesn't, neither does it effect the result of the code. It does though a redundant declaration of the type name.

File: src/Compilers/VisualBasic/Portable/Errors/DiagnosticFormatter.vb [hunk 2]
```diff
@@ -14,12 +14,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Sub
 
         Friend Overrides Function FormatSourceSpan(span As LinePositionSpan, formatter As IFormatProvider) As String
-            Return String.Format("({0}) ", span.Start.Line + 1)
+            Return "(" & span.Start.Line + 1 &") "
```
Comment (AlekseyTs) [inline src/Compilers/VisualBasic/Portable/Errors/DiagnosticFormatter.vb]: >span.Start.Line + 1 [](start = 25, length = 19)

Should probably call ToString explicitly to avoid an invocation of VB runtime helper.

Comment (AdamSpeight2008) [inline src/Compilers/VisualBasic/Portable/Errors/DiagnosticFormatter.vb]: @AlekseyTs  Done.

**Comment by AlekseyTs** [comment]
LGTM

**Comment by jcouv** [comment]
test windows_eta_open_prtest please

**Comment by jcouv** [comment]
Thanks @AdamSpeight2008. Merged.
